tests/utilities/test_ewma.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
+import numpy as np
+
+from fastats.utilities.ewma import ewma
+
+
+def sample_data_series():
```

Comment:
nice! would be good to parameterise this, as in `sample_data_series(n, dtype='float')`, or use hypothesis\reasonable to generate the data